### DISCUSSION TITLE
Trading API parameter

### DISCUSSION THREAD

**Discussion by dhoaison** [cloudbet/docs#47] [Q&A] [answered]
Hi everyone. I am testing betting using cloudbet API. But there is a parameter name **customerReference**. I read documents but have no clue about it. Could you please explain this parameter? Thank you.
![image](https://user-images.githubusercontent.com/32816961/208852909-fc835d3d-c850-4dc0-b8c2-f4196f42450e.png)

**Comment by kgravenreuth** [comment, marked as answer]
Hi @dhoaison 

Thanks for raising this issue. For now, please feel free to skip the `customerReference` field in your requests and they should go through without issues. We will further clarify this field in the docs soon.

Let me know if you see any other issues.

Cheers,
Klaus Gravenreuth
Product & Engineering @ Cloudbet
